# Make sure with_bonjour value gets set by corresponding AC_ARG_WITH
# Gentoo: https://bugs.gentoo.org/show_bug.cgi?id=283332
--- a/libgphoto2_port/configure.ac	2010-01-07 00:25:53.000000000 +0100
+++ b/libgphoto2_port/configure.ac	2010-01-07 00:25:43.000000000 +0100
@@ -161,7 +161,6 @@
 dnl ---------------------------------------------------------------------------
 dnl Apple Bonjour / mDNSResponder multicast DNS queries.
 dnl ---------------------------------------------------------------------------
-with_bonjour=no
 AC_ARG_WITH([bonjour],
 	AS_HELP_STRING([--without-bonjour], [whether to exclude Bonjour PTP/IP autodetection]),
 	,
